c2corg_api/views/feed.py

```diff
@@ -276,11 +276,18 @@ def load_feed(changes, lang):
     if not changes:
         return {'feed': []}
 
-    # TODO deal with redirected documents
-
     documents_to_load = get_documents_to_load(changes)
     documents = load_documents(documents_to_load, lang)
 
+    # only return changes for which the document/user could be loaded
+    changes = [
+        c for c in changes
+        if documents.get(c.user_id) and documents.get(c.document_id)
```

Comment:
Redirected documents are not included in the response to `load_documents` (one line above).